src/status_im/ui/screens/discover/components/views.cljs

```diff
@@ -17,7 +17,7 @@
    [react/text {:style      styles/title-text
                 :font       :medium}
     (i18n/label label-kw)]
-   [react/touchable-highlight {:on-press action-fn}
+   [react/touchable-highlight {:on-press (if active? action-fn #())}
```

Comment:
`when` should work fine?

Reply:
Might need restructuring of code as `(nil)` would be an error, no?. Agree it'd be better, though.

Reply:
Or simply `(when active? {:on-press action-fn})`

Reply:
`[react/touchable-highlight nil` works?

Reply:
IMO, the "right" way to do that is to add a custom property, e.g. `enabled?` or `active?` to touchable-highlight, and tbh I haven't even checked if it exists. I'm pretty sure this is not the only time we'll need to disable actions under certain conditions.

Reply:
But let's do that at some other occasion. The code above definitely works and it's pretty straightforward to read.

Reply:
@oskarth I believe it does?
@goranjovic You are right actually and it exists: https://facebook.github.io/react-native/docs/touchablewithoutfeedback.html#disabled  (parent class for `touchable-highlight`)

Reply:
@jeluard Are you cool with it as it is, i.e. can we move it forward? We can always re-visit this when we tackle with components after devcon?

Reply:
Did you gave `disabled` a try? Not a big deal but if we can remove us some future debt :)

Reply:
@jeluard done, works with disabled. Or rather, doesn't work ;)

Reply:
:+1: